contracts/contracts/core/vault/src/main.sw

```diff
@@ -1469,13 +1468,7 @@ fn _liquidate_position(
         available_collateral = available_collateral - position_fee;
     }
 
-    // TODO should we handle somehow the funding rate?
-    log(UpdatePnl {
-        key: position_key,
-        has_profit,
-        delta: pnl_delta,
-    });
-
+    // TODO should we handle somehow the funding rate and pnl?
```

Comment:
Maybe let's create a ticket for that in Jira, and delete this TODO comment.

Reply:
ok